### ISSUE TITLE
Faulty core fixes: Clash between PEAR.Functions.FunctionCallSignature.Indent and Generic.WhiteSpace.ScopeIndent.Incorrect

### ISSUE THREAD

**Issue by pento** [WordPress-Coding-Standards/WordPress-Coding-Standards#1000]
Given the following code:

```php
<?php
function foo() {
	?>
		<div><?php
		echo bar(
			$baz,
			$bat
		); ?></div>
	<?php
}
```

`phpcbf` fails to complete, as these two rules swap the indenting of the closing bracket of `bar()` between tabs and spaces.

This can be seen in Core in `src/wp-admin/includes/class-walker-nav-menu-edit.php`.

**Comment by pento** [comment]
Note: it appears that `PEAR.Functions.FunctionCallSignature.Indent` is the correct rule.

**Comment by pento** [comment]
`/src/wp-admin/setup-config.php` may be having issues with this bug, too. It has the same characteristics - nested PHP code embedded in HTML - but that might not be all that's going wrong with that file.

**Comment by jrfnl** [comment]
The problem here is that the embedded PHP is multi-line without the open/close PHP tags on their own line.

If they were, the fixer would finish without problem.

Adding the `Squiz.PHP.EmbeddedPhp` which has recently been added to `WordPress-Extra` in #991 to fix #465 to `WordPress-Core` would solve the issue.

The resulting fixed code would in that case look like:
```php
function foo() {
	?>
		<div>
		<?php
		echo bar(
			$baz,
			$bat
		);
		?>
		</div>
	<?php
}
```

@pento The handbook does not say anything specific about how PHP open/close tags should be layed out, so if we would decide to add the rule to `WordPress-Core`, I believe the handbook would need to be adjusted to make it explicit that that's part of the standard.

What do you think ?

**Comment by pento** [comment]
Huh, interesting. I never noticed that this was missing.

Yes, we can add it to the handbook, it's been a defacto standard for as long as I remember.

**Comment by jrfnl** [comment]
In that case, let me know when the handbook has been adjusted & I'll move the sniff from `Extra` to `Core`.

**Comment by jrfnl** [comment]
FYI: a branch to fix this has already been pushed, just waiting for some final tweaks to the handbook before pulling it.
